src/form-generator/form-handler-one-step.ts

```diff
@@ -6,66 +6,166 @@ import { FormRequiredFieldsParagraph } from "../form-element-creator/form-requir
+  };
+
+  createSendButtonElement = (containerForm: HTMLElement, apiKeyWeb3Forms: string): HTMLElement | undefined => {
+    const sendButtonConfig = this.getButtonConfig("send");
+    if (sendButtonConfig) {
+      return this.createButtonElement(sendButtonConfig, async () => {
+        if (this.validateForm()) {
+          await this.handleFormSubmissionWeb3Forms(apiKeyWeb3Forms, containerForm);
+        }
+      });
+    }
+    return undefined;
+  };
+
+  createButtonElement = (buttonConfig: ButtonConfig, clickHandler: () => void): HTMLFormElement => {
+    return new ButtonCreation().create(buttonConfig.value, clickHandler, buttonConfig.class);
+  };
+
+  getButtonConfig = (buttonName: string): ButtonConfig | undefined => {
+    return this.config.buttons.find((button: ButtonConfig) => button.name === buttonName);
   };
 
   validateForm = (): boolean => {
     const form = document.querySelector("form") as HTMLFormElement;
     return this.formValidator.validateForm(form);
   };
+
+  handleFormSubmissionWeb3Forms = async (apiKeyWeb3Forms: string, containerForm: HTMLElement): Promise<void> => {
+    try {
+      const formData = this.prepareFormDataForWeb3Forms();
+      const response = await this.fetchFormDataToWeb3Forms(apiKeyWeb3Forms, formData);
+      const result = await response.json();
+
+      if (result.success) {
+        this.handleSuccessfulSubmission(containerForm);
+      } else {
+        alert(`Form submission failed: ${result.message}`);
+      }
+    } catch (error) {
+      this.handleSubmissionErrorWeb3Forms(error);
+    }
+  };
+
+  prepareFormDataForWeb3Forms = (): FormData => {
+    const form = document.querySelector("#contact-form") as HTMLFormElement;
+    const formData = new FormData(form);
+    formData.delete('Password');
+    formData.delete('Confirm Password');
+    return formData;
+  };
+
+  fetchFormDataToWeb3Forms = (apiKeyWeb3Forms: string, formData: FormData): Promise<Response> => {
+    const formObject: { [key: string]: any } = {};
+    formData.forEach((value, key) => {
+      formObject[key] = value;
+    });
+
+    const API_URL_WEB3FORMS = "https://api.web3forms.com/submit";
+    return fetch(API_URL_WEB3FORMS, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+      },
+      body: JSON.stringify({ ...formObject, access_key: apiKeyWeb3Forms }),
+    });
+  };
+
+  handleSuccessfulSubmission = (containerForm: HTMLElement): void => {
+    const successMessageElement = this.formSuccessMessageHandler.createPopupWithSuccessMessage();
+    if (successMessageElement) {
+      containerForm.appendChild(successMessageElement);
+    }
+    console.log("Form submitted successfully!");
+    (document.querySelector("#contact-form") as HTMLFormElement).reset();
+  };
+
+  handleSubmissionErrorWeb3Forms = (error: unknown): void => {
+    if (error instanceof Error) {
+      alert(`An error occurred: ${error.message}`);
```

Comment:
rzucanie error.msg w alercie to zle podejscie. Po co userowi widziec kod bledu? Chyba ze to Twoj kod pod tytulem costam sie zepsulo musisz poprawic to czy tamto. Jezeli to dla programisty to console.error tylko i wylacznie bo pani jadzia z warzywniaka nie zrozumie co to jest inaczej.
Dodatkowow wszystkie komunikaty musza uwzgledniac jezyk wybrany ( te komunikatyt ktore userom daejesz nie konsola dla deva)